caddyhttp/httpserver/plugin.go

```diff
@@ -437,6 +437,7 @@ var directives = []string{
 	"hugo",        // github.com/hacdias/caddy-hugo
 	"mailout",     // github.com/SchumacherFM/mailout
 	"awslambda",   // github.com/coopernurse/caddy-awslambda
+	"maxrequestbody",
```

Comment:
Also I wonder if it should be called `max_req_body` or something a little shorter (and readable).